display-brightness-ddcutil@themightydeity.github.com/indicator.js

```diff
@@ -355,7 +354,7 @@ export const SingleMonitorSliderAndValueForQuickSettings = GObject.registerClass
             clearTimeout(this._timer);
     }
 
-    destroy() {
+    destory() {
```

Comment:
No sure why changing this to de-story, fixes Quicksetting slider not destorying when extension is disabled on when switch to systemtray panel mode.